app/components/links/TextLink.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link";
+import React from "react";
+
+export const TextLink = ({ url, text }: { url: string; text: string }) => {
```

Comment:
Use React.FC (Function Component) type for your component to indicate that it's a functional component.
```suggestion
const TextLink: React.FC<TextLinkProps> = ({ url, text, className }) => {
```

Reply:
make it as react functional component